### PR TITLE
fix(content): add `weapon` to Incipias ships

### PR DESCRIPTION
**Pull request by TheGiraffe3**
**Balance/Bug Fix**

## Summary
Adds a missing `weapon` to Incipias ships so that they explode properly.

## Screenshots
None.

## Testing Done
Not yet - I'm not a murderer.

## Save File
This save file can be used to test these changes:
Any save file that has a Jump Drive.

### CODE REVIEW

**Review by RisingLeaf — APPROVED** [review]
No balance approval, but good catch , reminder to me to create an explosion effect that matches metallic hydrogen

File: data/incipias/incipias ships.txt [hunk 1]
```diff
@@ -34,6 +34,11 @@ ship "Venta"
 		"engine capacity" 130
 		"gaslining" 1
 		"fuel capacity" 200
+		weapon
+			"blast radius" 20
+			"shield damage" 200
+			"hull damage" 100
+			"hit force" 300
```
Comment (Quantumshark) [inline data/incipias/incipias ships.txt:41]:
```suggestion
			"blast radius" 224
			"shield damage" 2240
			"hull damage" 1120
			"hit force" 3360
```

File: data/incipias/incipias ships.txt [hunk 2]
```diff
@@ -86,6 +91,11 @@ ship "Nimbus"
 		"engine capacity" 85
 		"gaslining" 1
 		"fuel capacity" 200
+		weapon
+			"blast radius" 10
+			"shield damage" 100
+			"hull damage" 50
+			"hit force" 150
```
Comment (Quantumshark) [inline data/incipias/incipias ships.txt:98]:
```suggestion
			"blast radius" 100
			"shield damage" 1000
			"hull damage" 500
			"hit force" 1500
```

**Review by Quantumshark — COMMENTED** [review]
Numbers based on the system described in https://github.com/endless-sky/endless-sky/wiki/CreatingShips

image

Your numbers for the Gero, Nimbo Cirrus and Nimbo Stratus are already correct.

File: data/incipias/incipias ships.txt (hunk 2 above)
Comment (TheGiraffe3) [reply data/incipias/incipias ships.txt:98]: Guess I missed a zero here. Sorry!

**Pull request merged**